File: lab/python/tecnicas/inverter_variaveis.py
def versao1():
    n1, n2 = input().split()
    n1, n2 = int(n1), int(n2)

    temp = n1
    n1 = n2
    n2 = temp

    print(n1, n2)


def versao2():
    n1, n2 = map(int, input().split())
    n1, n2 = n2, n1
    print(n1, n2)


# versao1()
versao2()

File: lab/javascript/tecnicas/inverter_variaveis.js
const prompt = require('prompt-sync')();


function versao1(){
    let [n1, n2] = prompt().split(" ").map(Number);
    
    let temp = n1;
    n1 = n2;
    n2 = temp;

    console.log(n1, n2);
}

function versao2(){
    let [n1, n2] = prompt().split(" ").map(Number);
    [n2, n1] = [n1, n2];
    console.log(n1, n2);
}


// versao1();
versao2();
